packages/ingest-mongodb-public/package.json

```
@@ -58,6 +62,9 @@
     "mongodb-rag-ingest": "*",
     "node-fetch": "^2.7.0",
     "striptags": "^3.2.0",
+    "turndown": "^7.2.0",
+    "turndown-plugin-gfm": "^1.0.2",
+    "xml2js": "^0.6.2",
     "yaml": "^2.6.0"
   }
-}
+}
```

```suggestion
}

```

keep the newline at end pls